### DISCUSSION TITLE
Help with changing recipes and textures

### DISCUSSION THREAD

**Discussion by Bugheadben** [FabricMC/fabric-api#2838] [Mod Dev Support]
So im new to modding, and i found this really cool mod than i want to use for my world, but the recipes are a bit odd and i want to change one of the textures, but when i do both in intellij and try to get my JAR file, it is just filled with random fabric files and nothing from the mod, someone help please lol 
discord is Bugheadben#6906 if you want to dm me there, i can provide the mod page and the textures im trying to use
version is 1.19.2

![image](https://user-images.githubusercontent.com/118641320/211179399-9ce765f4-630b-4011-b89a-21470547db20.png)
![image](https://user-images.githubusercontent.com/118641320/211179429-88b70853-f6e6-4c63-b7c9-5e367b614463.png)

**Comment by KosmX** [comment]
How do you build the `jar`?  
`./gradlew build` and use `./build/libs/[mod].jar`

**Reply by Bugheadben** [reply]
I know how to build the jar, its just its filled with random files
